SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -140,22 +151,40 @@ - (void)disconnect {
 
 #pragma mark - Creating Session Streams
 
-- (void)sdl_establishSession {
+- (BOOL)sdl_tryConnectAccessory:(EAAccessory *)accessory {
```

Comment:
I believe that's actually the exception rather than the rule. For example, in the `NSFileManager` APIs, the method `createFileAtPath:contents:attributes` returns a BOOL for success, as does `createDirectoryAtURL:blahblah:`, and many others. `tryLock` is the only one I've ever heard of that uses `try`.